cypress/e2e/chi-custom-elements/common/data-picker-common.cy.js

```diff
@@ -0,0 +1,32 @@
+export const DATE_PICKER_MINUTESSTEP_VALUES = [
```

Comment:
There's a typo in the filename! datE-...  😸

Reply:
Ok, I change it in the next commit. Thanks. 

Also, there will be more changes in order to improve the code.